static/js/transform-facets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * A Facets component transformFacets function which applies the configuration
```

Comment:
It might be beneficial to use the verbiage "transform" here. This function transforms the Facets.fields component configuration provided by the Hitchhikers into a `transformFacets` function that can be understood by the SDK.

Reply:
that works